spec/fluent.ebnf

```diff
@@ -30,11 +30,17 @@ Value               ::= Pattern
 Pattern             ::= PatternElement+
 VariantList         ::= blank? "{" variant_list blank? "}"
 PatternElement      ::= TextElement
+                      | block_text
                       | Placeable
-                      | (blank_block blank_inline? Placeable)
+                      | block_placeable
+
+/* TextElement and Placeable can occur inline or as block.           *
+ * Text needs to be indented and start with a non-special character. *
+ * Placeables can start at the beginning of the line or be indented. */
 TextElement         ::= text_char+
-                      | text_cont
+block_text          ::= blank_block blank_inline indented_char text_char*
```

Comment:
Content-wise, this is spot on. I just think we can group this a bit differently. If we use `TextElement` as a name of a production, than `indented_char text_char*` hides the fact that it's a `TextElement` as well. We could do `indented_char TextElement*`, but that is also misleading: it hides the fact that `indented_char` is a `TextElement` as well.

```
/* TextElement and Placeable can occur inline or as block.
 * Text needs to be indented and start with a non-special character.
 * Placeables can start at the beginning of the line or be indented. */
PatternElement      ::= inline_text
                      | indented_text
                      | inline_placeable 
                      | indented_placeable 

inline_text ::= text_char+
indented_text ::= blank_block blank_inline indented_char inline_text*
inline_placeable ::= "{" blank? (SelectExpression | InlineExpression) blank? "}"
indented_placeable ::= blank_block blank_inline? inline_placeable 
```

Yay, symmetry!